src/main/java/net/minecraftforge/network/PlayMessages.java

```diff
@@ -122,23 +134,36 @@ public static SpawnEntity decode(FriendlyByteBuf buf)
                     buf.readDouble(), buf.readDouble(), buf.readDouble(),
                     buf.readByte(), buf.readByte(), buf.readByte(),
                     buf.readShort(), buf.readShort(), buf.readShort(),
-                    buf
+                    readSpawnDataPacket(buf)
                     );
         }
 
+        private static FriendlyByteBuf readSpawnDataPacket(FriendlyByteBuf buf) {
+            final int count = buf.readVarInt();
+            if (count > 0) {
```

Comment:
Formatting: braces on newlines.
```suggestion
        private static FriendlyByteBuf readSpawnDataPacket(FriendlyByteBuf buf)
        {
            final int count = buf.readVarInt();
            if (count > 0)
            {
```